README.md

```diff
@@ -82,12 +82,22 @@ createNav({ maxWidth: '80rem' });
 
 To build the JS into the `/dist` folder, run:
 
-##implest way to run the site locally is to first [install Docker](https://docs.docker.com/engine/installation/) (on Linux you may need to [add your user to the `docker` group](https://docs.docker.com/engine/installation/linux/linux-postinstall/)), and then use the `./run` script:
+`./run build`
```

Comment:
I think a code block is more practical as you get the copy to clipboard button

Reply:
today I learned! will update